7_react/react2/week1/user-search/src/App.js

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+import { GithubProvider, useGithubContext } from './components/GithubContext';
+
+const App = () => {
+  const { users, loading, error, searchUsers, clearUsers } = useGithubContext();
+  const [query, setQuery] = useState('');
+
+  const handleSearch = (e) => {
+    const value = e.target.value;
+    setQuery(value);
+
+    if (value.trim() !== '') {
```

Comment:
You could remove the `!== ''` because an empty string is treated as false in JavaScript. See https://developer.mozilla.org/en-US/docs/Glossary/Falsy